spec/hosts/environment_spec.rb

```diff
@@ -4,8 +4,10 @@
   context 'without an explicit environment setting' do
     it { should contain_file('environment').with_path('rp_env') }
   end
-  context 'when specifying an explicit environment' do
+  # Broken on ~> 3.8.5 since PUP-5522
+  context 'when specifying an explicit environment', :unless => (Puppet.version >= '3.8.5' && Puppet.version.to_i < 4) do
```

Comment:
Sorry, this stumped me without monkey patching (which perhaps belongs in puppetlabs_spec_helper?).